### PR TITLE
make.bat: add default msvc2015 config when there is no vswhere.exe

### PR DESCRIPTION
**Pull request by yuyi98**
This PR add default msvc2015 config when there is no `vswhere.exe`.

### CODE REVIEW

File: make.bat [hunk 1]
```diff
@@ -59,8 +59,10 @@ for /f "usebackq tokens=*" %%i in (`"%VsWhereDir%\Microsoft Visual Studio\Instal
 	set InstallDir=%%i
 )
 
-if exist "%InstallDir%\Common7\Tools\vsdevcmd.bat" (
-	call "%InstallDir%\Common7\Tools\vsdevcmd.bat" -arch=%HostArch% -host_arch=%HostArch% -no_logo
+if exist %InstallDir%\Common7\Tools\vsdevcmd.bat (
```
Comment (spytheman) [inline make.bat:62]: Previously it checked `if exist "..." (`, I think the quoting was needed because InstallDir could contain spaces, no?

File: make.bat [hunk 2]
```diff
@@ -59,8 +59,10 @@ for /f "usebackq tokens=*" %%i in (`"%VsWhereDir%\Microsoft Visual Studio\Instal
...
+	call %InstallDir%\Common7\Tools\vsdevcmd.bat -arch=%HostArch% -host_arch=%HostArch% -no_logo
+) else if exist "%VsWhereDir%/Microsoft Visual Studio 14.0/Common7/Tools/vsdevcmd.bat" (
```
Comment (spytheman) [inline make.bat:64]: I am not sure BAT files support else if.
https://stackoverflow.com/questions/11081735/how-to-use-if-else-structure-in-a-batch-file

File: make.bat [hunk 3]
```diff
@@ -80,15 +82,13 @@ if %ERRORLEVEL% NEQ 0 (
 	echo V failed to build itself with error %ERRORLEVEL%
 	rd /s /q vc
 	del v_old.exe
-	del v2.pdb
```
Comment (spytheman) [inline make.bat:83]: why is that deleted, does v not produce v2.pdb anymore?

Comment (yuyi98) [reply make.bat:83]: No v2.pdb generated.

File: make.bat (hunk 2 above)
Comment (yuyi98) [reply make.bat]: Yes, it support.

File: make.bat (hunk 1 above)
Comment (yuyi98) [reply make.bat]: Yes, I made a mistake.

File: make.bat (hunk 3 above)
Comment (spytheman) [reply make.bat:83]: even after `v up` and `v self` ?

Comment (yuyi98) [reply make.bat:83]: Yes, there is v2.exe and v3.exe temp files before.
